src/System.Net.WebSockets.Client/tests/CancelTest.cs

```diff
@@ -117,23 +117,52 @@ public class CancelTest : ClientWebSocketTestBase
                 await cws.CloseOutputAsync(WebSocketCloseStatus.NormalClosure, "CancelShutdown", cts.Token);
             }, server);
         }
+        
+        [OuterLoop]
+        [ConditionalTheory(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
+        public async Task ReceiveAsync_CancelThenReceive_ThrowsOperationCanceledException(Uri server)
+        {
+            using (ClientWebSocket cws = await WebSocketHelper.GetConnectedWebSocket(server, TimeOutMilliseconds, _output))
+            {
+                var recvBuffer = new byte[100];
+                var segment = new ArraySegment<byte>(recvBuffer);
+                var cts = new CancellationTokenSource();
 
-        [OuterLoop] // TODO: Issue #11345
```

Comment:
The TODO for #11345 should not have been removed.  This test still should be OuterLoop and still should have the TODO.